src/components/TodoList.tsx

```diff
@@ -83,7 +95,7 @@ export const TodoList = ({ item, deleteFn, id, handleChange }) => {
                 <Button
                   data-testid="btn-edit"
                   color="info"
-                  onClick={() => handleEdit(id)}
+                  onClick={() => handleEdit()}
```

Comment:
It can be changed to this.
```suggestion
                  onClick={handleEdit}
```